certified-connectors/Encodian/apiDefinition.swagger.json

```diff
@@ -1648,43 +1659,43 @@
           {
             "in": "body",
             "name": "operation",
-            "description": "Request object, refer to schema definition: DtoExtractTextRegion",
+            "description": "Request object, refer to schema definition: DtoSplitPdfByText",
             "schema": {
-              "$ref": "#/definitions/DtoExtractTextRegion"
+              "$ref": "#/definitions/DtoSplitPdfByText"
             }
           }
         ],
         "responses": {
           "200": {
             "description": "Success",
             "schema": {
-              "$ref": "#/definitions/DtoResponseExtractTextRegion"
+              "$ref": "#/definitions/DtoResponseSplitPdfByText"
             }
           },
           "202": {
             "description": "Accepted",
             "schema": {
-              "$ref": "#/definitions/DtoResponseExtractTextRegion"
+              "$ref": "#/definitions/DtoResponseSplitPdfByText"
             }
           },
           "400": {
             "description": "Bad request",
             "schema": {
-              "$ref": "#/definitions/DtoResponseExtractTextRegion"
+              "$ref": "#/definitions/DtoResponseSplitPdfByText"
             }
           }
         },
         "x-ms-visibility": "important",
-        "summary": "Extract Text from Regions",
-        "description": "Extract text using OCR from 70+ files types using specified regions - https://support.encodian.com/hc/en-gb/articles/360015612353-Extract-Text-from-Regions"
+        "summary": "Split PDF by Text",
+        "description": "Split a PDF Document by text into multiple PDF Documents - https://support.encodian.com/hc/en-gb/articles/360012726397-Split-PDF-by-Text"
       }
     },
-    "/api/v1/Core/AddHtmlHeaderFooter": {
+    "/api/v1/Core/SplitPdfByBarcode": {
```

Comment:
Can you please ensure that no existing flows will break? Please see the below error. 
##[error]The output field has been changed from required to optional. Existing flows may rely on this field as required, they may be broken. ValuePath: operation/SplitPdfByBarcode/responses/200/schema/documents/required Source: NewSwagger

Reply:
@vmanoharas - we handle this within the API this can be resolved :)